opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/aggregation/dsl/MetricAggregationBuilder.java

```diff
@@ -176,6 +198,24 @@ public MetricAggregationBuilder(ExpressionSerializer serializer) {
     return Pair.of(aggregationBuilder, parser);
   }
 
+  /**
+   * Make {@link CardinalityAggregationBuilder} for distinct count aggregations.
+   */
+  private Pair<AggregationBuilder, MetricParser> make(CardinalityAggregationBuilder builder,
```

Comment:
what is the difference between existing make? can we merge them?

Reply:
The difference is the builder type. I think it's fine to remain them as overwritten methods to get rid of judging instance type.